dsp-repository/v1/dsp-repository.shacl.ttl

```diff
@@ -26,32 +26,670 @@
+        sh:name "is Part of" ;
+        sh:description "The project to which the data set belongs"@en ;
+        sh:class dsp-repo:Project ;
+        sh:maxCount 1 ;
+        sh:minCount 1 ;
+    ] ;
+    sh:property [
+        sh:path dsp-repo:hasQualifiedAttribution ;
+        sh:name "Qualified attribution" ;
+        sh:description "Persons/Organization involved in the creation of the dataset"@en ;
+        sh:class prov:Attribution ;
+        sh:minCount 1 ;
+    ] ;
+    sh:property [
+        sh:path dsp-repo:hasDateCreated ;
+        sh:name "Date created" ;
+        sh:description "Creation date of the dataset"@en ;
+        sh:datatype xsd:dateTime   ;
+        sh:maxCount 1 ;
+    ] ;
+    sh:property [
+        sh:path dsp-repo:hasDateModified ;
+        sh:name "Date modified" ;
+        sh:description "Modification date of the dataset"@en ;
+        sh:datatype xsd:dateTime   ;
+        sh:maxCount 1 ;
+    ] ;
+    sh:property [
+        sh:path dsp-repo:hasDistribution ;
+        sh:name "Distribution" ;
+        sh:description "A downloadable form of this dataset, at a specific location, in a specific format"@en ;
+        sh:class schema:DataDownload ;
+        sh:maxCount 1 ;
+    ] ;  
+    sh:closed true ;
+    sh:ignoredProperties (rdf:type) .
+
+### --- Shape of dsp-repo:Person ---
+dsp-repo:PersonShape
+    a sh:NodeShape ;
+    sh:targetClass dsp-repo:Person ;
+    sh:property [
+        sh:path dsp-repo:hasGivenName ;
+        sh:name "Given name" ;
+        sh:description "Given name of the person"@en ;
+        sh:datatype xsd:string ;
+        sh:minCount 1 ;
+    ] ;
+     sh:property [
+        sh:path dsp-repo:hasFamilyName ;
+        sh:name "Family name" ;
+        sh:description "Family name of the person"@en ;
+        sh:datatype xsd:string ;
+        sh:minCount 1 ;
+    ] ;
+     sh:property [
+        sh:path dsp-repo:hasEmail ;
+        sh:name "E-mail" ;
+        sh:description "E-mail address of the person"@en ;
+        sh:datatype xsd:string ;
+        sh:maxCount 2 ;
+    ] ;
+     sh:property [
+        sh:path dsp-repo:hasAddress ;
+        sh:name "Address" ;
+        sh:description "Postal address of the person"@en ;
+        sh:datatype xsd:string ;
+        sh:maxCount 1 ;
+    ] ;
+     sh:property [
+        sh:path dsp-repo:isMemberOf ;
+        sh:name "Member of" ;
+        sh:description "Affiliation of the person"@en ;
+        sh:class dsp-repo:Organization ;
+        sh:minCount 1 ;
+    ] ;
+     sh:property [
+        sh:path dsp-repo:hasJobTitle ;
+        sh:name "Job title" ;
+        sh:description "Position / Job title of the person"@en ;
+        sh:datatype xsd:string ;
+        sh:minCount 1 ;
+    ] ;
+     sh:property [
+        sh:path dsp-repo:hasIdentifier ;
+        sh:name "Identifier" ;
+        sh:description "Identifier of the person"@en ;
+        sh:class schema:URL ;
+        # sh:or ( [ sh:datatype xsd:string ] [ sh:nodeKind sh:IRI ] ) ;
```

Comment:
I think we need to create a class for : 
- `Identifier`
- `License`
- `TemporalCoverage`
- `SpatialCoverage`
and maybe `QualifiedAttribution` but this is not entirely clear to me if we want to create data such as in the `example-metadata.ttl` file below.

As I understand it:
- we want to provide an IRI
- *and* we want to give information regarding the provider of that IRI by using a string.  

I don't know how we could do this and also define SHACL constraints on if we don't use classes for these. This is basicaly what we did for `Grant` and I think we need this in these cases too.

Reply:
no need for more classes, please see: 

- https://schema.org/identifier

- https://schema.org/license

- `  sh:property [

        sh:path dsp-repo:hasDiscipline ;
        sh:name "Discipline" ;
        sh:description "Discipline and research fields from UNESCO nomenclature: https://skos.um.es/unesco6/?l=en or from http://www.snf.ch/SiteCollectionDocuments/allg_disziplinenliste.pdf"@en ;
        sh:or ( [ sh:datatype xsd:string ] [ sh:class schema:URL ] ) ;
        sh:minCount 1 ;
    ] ;`

- https://schema.org/temporalCoverage

- https://schema.org/spatialCoverage

- https://www.w3.org/TR/prov-o/#qualifiedAttribution